style_guide.md

```diff
@@ -0,0 +1,18 @@
+# Style Guide
+
+Please follow the guidelines stated here when contributing to this repository.
+
+## Spacing
+
+- Always use tabs. Spaces suck.
+- Avoid trailing spaces at the end of lines.
+- Avoid trailing empty lines at the end of files.
+- Surround operators with a single space (except for when using `=` in parameter lists).
+- Leave two empty lines between functions.
+- Optionally leave one empty line between logical blocks of code in the same function.
+
```

Comment:
Suggest installing Prettier extension?

I just let them decide how to format the code lol

Reply:
I think they use spaces... will check out prettier, tho I currently use the Python extension for formatting which uses autopep8

Reply:
IMO doesn't matter as long as its readable. Spaces are fine if it's consistent.

Reply:
Actually just checked, prettier doesn't have Python support. They dont reference a python plugin in their docs either. We may as well use autopep8 which is the official one. I dont think it converts tabs or spaces tho

Reply:
We're using Black (but with tabs)